src/main/kotlin/no/nav/familie/tilbake/behandling/domain/Behandling.kt

```diff
@@ -59,6 +59,8 @@ data class Behandling(@Id
 
     val sisteResultat get() = resultater.maxByOrNull { it.sporbar.endret.endretTid }
 
+    val sisteÅrsak get() = årsaker.firstOrNull()
```

Comment:
Du må vel sortere årsaker for å få den siste?

Reply:
Jeg tror behandling ikke kan ha flere årsaker. Så det burde være greit.

Reply:
Ok. Da burde vi kanskje endre årsaker fra Set<Årsak> til Årsak, men det kan vi ta i en egen PR.